bindings/nodejs/lib/Account.ts

```diff
@@ -515,7 +533,7 @@ export class Account {
      * @returns The incoming transactions with their inputs.
      */
     async incomingTransactions(): Promise<
-        [string, [ITransactionPayload, IOutputResponse[]]][]
+        [string, IncomingTransactionData][]
```

Comment:
I think the IOutputResponse needs to stay, it's the inputs of the transaction if still available

Reply:
I just tried to follow what you returned inside the rs code (https://github.com/iotaledger/wallet.rs/blob/feat/update-account-methods/src/message_interface/response.rs#L29). I think Thibault changed it to return `IOutputMetadataResponse` and I created my own type to make it a bit more readable.

Reply:
Ah sorry missed that it's a new type